amy/autoemails/admin.py

```diff
@@ -68,14 +69,19 @@ class RQJobAdmin(admin.ModelAdmin):
         "mail_status",
         "event_slug",
         "recipients",
+        "interval",
+        "result_ttl",
     ]
     date_hierarchy = "created_at"
     readonly_fields = [
         "scheduled_execution",
+        "action_name",
         "status",
         "mail_status",
         "event_slug",
         "recipients",
+        "interval",
+        "result_ttl",
```

Comment:
There is a template (`amy/autoemails/templates/rqjob_preview.html`) for previewing RQJob email. Do you want to display these new fields there too?

Reply:
Added them as well. Thanks for the suggestion